git-land

```diff
@@ -161,6 +150,15 @@ head=$(git rev-parse HEAD)
   git merge --ff-only $head) || \
     exit_and_cleanup $? "Could not fast-forward merge $merge_branch into $target_branch"
 
+# create last "merge" commit with github tag to close PR
+if [ -n "$pr" ]; then
+  commit_message="Merge $merge_branch to $target_branch"
+
+  if ! (git commit -n --allow-empty -m "$commit_message"$'\n\n'"[close PR#$pr]"); then
```

Comment:
1. it's better to build the whole commit-message in a var above
2. I wonder if we can fetch pull-request's title/description from github…